packages/graphql-api/src/utils/context-builder.ts

```diff
@@ -9,6 +9,7 @@ export const context = (moduleName: string) => async (
   _: any,
   { injector }: ModuleSessionInfo
 ) => {
+  if (!req) { return; }
```

Comment:
You should return null here not to break typings

Reply:
Fixed!

Reply:
Hmm, it didn't work either

Reply:
Oh wait, we should return `{}` and make all fields of context interface optional.